docs/security/configuration/proxy.md

```diff
@@ -57,7 +57,6 @@ proxy_auth_domain:
   order: 0
   http_authenticator:
     type: proxy
-    challenge: false
```

Comment:
Hey @mmadoo did including the challenge lines cause problems in your configuration? My understanding is that you do want challenge set to false, so even if the default for an unspecified value is false, I think being explicit is still the way to go.

Reply:
I have re-test it and it works when challenge is set. I have rollback this change.